docs/compat-tokens-inventory.md

```
@@ -0,0 +1,37 @@
+### Compat tokens inventory (snapshot)
+
+対象: `--color-white-900`, `--color-black-600`, `--color-black-850`, `--color-white-850`（tokens.css除外）
+
+実行: `pnpm report:compat-tokens`
+
+出力例（抜粋）
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)